mypy/server/deps.py

```diff
@@ -153,13 +170,13 @@ def __init__(self,
     #   await
     #   protocols
     #   metaclasses
-    #   type aliases
     #   functional enum
     #   type variable with value restriction
 
     def visit_mypy_file(self, o: MypyFile) -> None:
         self.scope.enter_file(o.fullname())
         self.is_package_init_file = o.is_package_init_file()
+        self.add_type_alias_deps(self.scope.current_full_target())
```

Comment:
This should be `self.scope.current_target()`? `current_full_target()` can refer to an entire class (all method bodies) which we want to avoid if possible, since it's expensive.